src/extensions/default/DarkTheme/main.less

```diff
@@ -59,7 +59,7 @@
         color: #fff;
     }
     .inline-widget .CodeMirror-gutter-elt {
-        color: #767676;    
```

Comment:
@swmitra  there were some extra spaces after the semi colon, my editor is configured to string extra spaces.